pythoncalculator/multiply.py

```diff
@@ -0,0 +1,2 @@
+def multiply(x, y):
+    return x + y
```

Comment:
We're trying to multiply here, Paul, you silly goose!
```suggestion
    return x * y
```